composer.json

```diff
@@ -3,11 +3,10 @@
         "ext-curl": "*"
     },
     "require-dev": {
-        "phpunit/phpunit": "4.5.1"
+        "phpunit/phpunit": "~4.5,>=4.5.1"
```

Comment:
Using `^4.51` would be appropriate I think - https://getcomposer.org/doc/01-basic-usage.md#next-significant-release-tilde-and-caret-operators-